### PR TITLE
QOL update

### PR DESCRIPTION
**Pull request by weespin**
Faster transition speed
Show item stats on dropped items
Some typo fixes
Fixed text rendering behind player (Hitmarkers/Dropped items)
Small refactoring
Fixed meteor footshake

### CODE REVIEW

File: ChampionsOfForest.sln [hunk 1]
```diff
@@ -1,24 +1,18 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
```
Comment (Hazardu) [inline ChampionsOfForest.sln:2]: can you undo all changes to ChampionsOfForest.sln?

File: Items/ItemDataBase_ItemDefinitions.cs [hunk 2]
```diff
@@ -4604,7 +4604,7 @@ public static void PopulateItems()
...
+					description = "What makes me a good demoman? If I were a bad demoman, I wouldn't be sittin' here discussin' it with you, now would I?! LET'S DO IT! Not one of you's gonna survive this! One crossed wire, one wayward pinch of potassium chlorate, one errant twitch, and KA-BLOOIE! I got a manky eye. I'm a black Scottish cyclops. They got more fecking sea monsters in the great Lochett Ness than they got the likes of me. So! T'all you fine dandies, so proud, so cocksure, prancin' about with your heads full of eyeballs... come and get me, I say! I'll be waitin' on you with a whiff of the old brimstone! I'm a Grimm bloody fable with an unhappy bloody end! Oh, they're going to have to glue you back together...IN HELL!",
```
Comment (Hazardu) [inline Items/ItemDataBase_ItemDefinitions.cs:4607]: lol

File: Items/ItemPickUp.cs [hunk 3]
```diff
@@ -69,37 +71,80 @@ private void OnGUI()
...
+					GUI.color = MainMenu.RarityColors[stat.Rarity];
+					//Name
+					statStyle.alignment = TextAnchor.UpperLeft;
+
+					GUI.Label(new Rect(r.x, r.y + titleHeight + (i * lineheight), r.width, r.height), statslabel, statStyle);
+					//Value
+					statStyle.alignment = TextAnchor.UpperRight;
+					GUI.Label(new Rect(r.x, r.y + titleHeight + (i * lineheight), r.width, r.height), statsvalue, statStyle);
+
+				}
```
Comment (Hazardu) [inline Items/ItemPickUp.cs:146]: good stuff, also gave me an idea to include some more info on item markers

**Review by Hazardu — CHANGES_REQUESTED** [review]
Please undo changes to .csproj and .sln

File: Items/ItemDataBase_ItemDefinitions.cs (hunk 2 above)
Comment (weespin) [reply Items/ItemDataBase_ItemDefinitions.cs:4607]: https://wiki.teamfortress.com/wiki/Meet_the_Demoman
🤷‍♂️

**Comment by weespin** [comment]
Done

**Pull request merged**